Civi/Api4/MailingJob.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4;
+
+/**
+ * Mailing job.
+ *
+ * Mailing job tracks the batching of CiviMails.
+ *
+ * @searchable none
+ * @see https://docs.civicrm.org/user/en/latest/email/what-is-civimail/
+ * @since 5.47
+ * @package Civi\Api4
+ */
+class MailingJob extends Generic\DAOEntity {
```

Comment:
Should we make `MailingJob` readonly?

Reply:
done

Reply:
@eileenmcnaughton the `@readonly` annotation isn't right. Instead you have to use the `ReadOnlyEntity` trait.

Reply:
@eileenmcnaughton ^

Reply:
@colemanw thanks - fixed